src/temp/ProjectInfo/SingleProjectInfoItem.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement } from 'react';
+import styles from './ProjectInfo.module.scss';
+
+interface Props {
+  title: ReactElement | string;
+  children: ReactElement;
+}
+
+const SingleProjectInfoItem = ({ title, children }: Props) => {
+  return (
+    <div className={styles.singleRowInfoContainer}>
+      <div className={styles.singleRowInfoContent}>
+        <div className={styles.singleRowInfoTitle}>{title}</div>
```

Comment:
So that we provide a semantically meaningful header tag to the title content instead of a div, how about adding a default tag of `h2` (see if `h2` is the appropriate level, or something lower is needed) if the `title` is a string? 

Providing a ReactElement will override the default.

Reply:
If I do this there will be an `h2` inside a `h2` element. This is the reason I went with fragments in the beginning as well. Below is the structure after the changes for `KeyInfo` component

`<div class="ProjectInfo_singleRowInfoContent__KdZ_q"><h2 class="ProjectInfo_singleRowInfoTitle__dtZ21"><h2 class="ProjectInfo_abandonmentTitle__obmbL">Abandonment<div aria-haspopup="true" style="cursor: pointer;"><svg </svg></div></h2></h2><p>Approx. 2005</p></div>`

Reply:
It's conditional, so you would not end up with `h2` inside `h2`.

The logic would be
`if title is a string, use a h2 tag, otherwise, use a div tag`